### ISSUE TITLE
qbittorrent started but can't access the webui, luckily, I registered magnet links in the past, so it was able to to load qbittorrent using the ingress link when I click on a magnet link, hope this provides more clues.

### ISSUE THREAD

**Issue by alexbelgium** [alexbelgium/hassio-addons#1135]
qbittorrent started but can't access the webui, luckily, I registered magnet links in the past, so it was able to to load qbittorrent using the ingress link when I click on a magnet link, hope this provides more clues.

Anyway to revert to an earlier version?

_Originally posted by @bilogic in https://github.com/alexbelgium/hassio-addons/issues/1127#issuecomment-1858010461_

**Comment by alexbelgium** [comment]
Hi, what is your log?

And what is the issue exactly? The thread you posted in was about mullvad vpn not working, thanks

**Comment by alexbelgium** [comment]
Have you tried without ssl?

**Comment by alexbelgium** [comment]
Hi, I'll revert to the previous upstream image middle of next week due to the many bugs with the new upstream image

**Comment by bilogic** [comment]
Sorry about that misreading the other issue.

I would summarize my issue as `qbitorrent running, but unable to load web-ui in HA`

Currently using `4.6.2-21_wireguard_openvpn`, my problem: the webui of qbitorrent cannot load when clicking on `open web-ui` button. It gave a 404 error. However, when I clicked on a magnet link, qbitorrent catches the link and loads (only itself, without HA)

I tried to reproduce the above moments ago, but the web ui loads successfully now.

My qbitorrent does not have a domain of its own, so relies only on HA's SSL. I will update again if I face it again.
